### PR TITLE
Add admin notifications

### PR DESCRIPTION
**Pull request by Frodotus**

### CODE REVIEW

File: django_ilmoitin/models.py [hunk 1]
```diff
@@ -19,9 +20,35 @@ class NotificationTemplate(TranslatableModel):
         verbose_name=_("type"),
         unique=True,
         db_column="type",
+        default="",
         choices=list(notifications.registry.items()),
     )
 
+    from_email = models.EmailField(
+        verbose_name=_("From email"), max_length=100, default="dev@hel.fi"
```
Comment (igordavydsson) [inline django_ilmoitin/models.py]: could you make the default come from `settings.DEFAULT_FROM_EMAIL` and mention this in the README? would probably be a cleaner approach for this.

File: django_ilmoitin/migrations/0002_auto_20190820_1204.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                related_name="_notificationtemplate_admins_to_notify_+",
+                to=settings.AUTH_USER_MODEL,
+                verbose_name="Admins to notify",
+            ),
+        ),
+        migrations.AddField(
+            model_name="notificationtemplate",
+            name="from_email",
+            field=models.EmailField(
+                default="dev@hel.fi", max_length=100, verbose_name="From email"
```
Comment (igordavydsson) [inline django_ilmoitin/migrations/0002_auto_20190820_1204.py]: forgot to update the migration? 🙂

Comment (Frodotus) [reply django_ilmoitin/migrations/0002_auto_20190820_1204.py]: So it seams :)

**Review by igordavydsson — APPROVED** [review]
Good stuff! Very welcome addition 👏

**Pull request merged**